clients/admin-ui/cypress/e2e/config-wizard.cy.ts

```diff
@@ -188,7 +188,7 @@ describe.skip("Config Wizard", () => {
       cy.wait("@postGenerate403");
       // Expect the custom message for this specific error.
```

Comment:
should this comment be updated too if it's now a generic message?

Reply:
This was updated, and I think there is a ticket for allowing more useful error messages on the Okta path coming up soon.